app/controllers/qa/linked_data_terms_controller.rb

```diff
@@ -120,6 +152,17 @@ def check_id_param
       end
     end
 
+    def check_uri_param
```

Comment:
@bmquinn Thanks for the review.  I fixed this and similar patterns for other params.  I squashed the commits, but the only change since your last review is to the params checks in linked_data_term_controller.rb.